content/module-reference/utility-modules/darkroom/color-assessment.md

```diff
@@ -6,15 +6,15 @@ tags:
 view: darkroom
 ---
 
-Assess colors and brightness in your image using ISO 12646:2008 recommended viewing conditions.
+Assess colors and brightness in your image under neutral viewing conditions to avoid perceptual issues.
 
 When developing an image, the way we perceive brightness, contrast and saturation is influenced by the surrounding ambient conditions. If an image is displayed against a dark background, this can have a number of adverse effects on our perception of that image:
 
 - Exaggeration of the perceived exposure makes the image seems brighter than it really is. This is nicely illustrated by the [Adelson checkerboard shadow effect](https://en.wikipedia.org/wiki/Checker_shadow_illusion).
 - A decrease in the perceived saturation in the image makes the colors seem less rich than they really are (the Hunt effect).
 - A decrease in the perceived contrast in the image makes the tones seem flatter than they really are (Bartleson-Breneman effect 3)
 
-The end result is that the final image can end up being too dark and overly-processed in terms of contrast and color saturation. To avoid this, the "ISO 12646:2008" standard makes some recommendations about the conditions under which the colors of an image should be assessed. The _color assessment_ module in the darkroom places a frame around the image to help the user better assess the colors in the image, along the lines of those recommendations.
+The end result is that the final image can end up being too dark and overly-processed in terms of contrast and color saturation. To avoid this, the _color assessment_ module in the _darkroom_ places a grey and white border around the image to help the user better assess the colors in the image
```

Comment:
We don't tend to italicise the names of the views in the user manual. I think we only generally italicise module names.